charts/linkerd2/README.md

```diff
@@ -0,0 +1,139 @@
+# Linkerd2 Helm Chart
+
+Linkerd is a *service mesh*, designed to give platform-wide observability,
+reliability, and security without requiring configuration or code changes.
+
+Linkerd is a Cloud Native Computing Foundation ([CNCF][cncf]) project.
+
+## Quickstart and documentation
+
+You can run Linkerd on any Kubernetes 1.12+ cluster in a matter of seconds. See
+the [Linkerd Getting Started Guide][getting-started] for how.
+
+For more comprehensive documentation, start with the [Linkerd
+docs][linkerd-docs].
+
+## Prerequisite: identity certificates
+
+The identity component of Linkerd requires setting up a trust anchor
+certificate, and an issuer certificate with its key. These need to be provided
+to Helm by the user (unlike when using the `linkerd install` CLI which can
+generate these automatically). You can provide your own, or follow [these
+instructions](https://linkerd.io/2/tasks/generate-certificates/) to generate new ones.
+
+## Installing the chart
+
+You must provide the certificates and keys described in the preceding section,
+and the same expiration date you used to generate the Issuer certificate.
+
+In this example we set the expiration date to one year ahead:
+
+```bash
+helm install \
```

Comment:
Use  `helm install --dep-up` to build the dependency.

Reply:
We'll end up including the dependency as we talked over slack, so the user won't have to deal with that.